package.json

```diff
@@ -17,6 +17,14 @@
   },
   "homepage": "https://github.com/imran5644/weatherApp#readme",
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
```

Comment:
Please add the following scripts:-
First is the dev script which is building the app in development mode

- [ ] ```"dev": "webpack --mode development"```

Then the build script which is building the app in production mode (for deployment)

- [ ] ```"build": "webpack --mode production"```

Although, you could do ```"build": "webpack"``` for both modes at the same time, but recommended separated.

<hr>

And the optional script is: 

- ```"start": "webpack serve"``` to start a live server for the app